README.md

```diff
@@ -1,27 +1,35 @@
-# Event Notification module
+# IBM Cloud Event Notifications module
+{: #EN-module}
```

Comment:
@hooznora This isn't cloud docs, so we don't need to add anchor IDs to the readme headings